ipaserver/install/server/install.py

```diff
@@ -618,6 +618,14 @@ def install_check(installer):
         # check addresses here, dns module is doing own check
         no_matching_interface_for_ip_address_warning(ip_addresses)
 
+    instance_name = "-".join(options.realm_name.split("."))
+    dirsrv = services.knownservices.dirsrv
+    if (options.external_cert_files and
+       dirsrv.is_installed(instance_name) and not
+       dirsrv.is_running(instance_name)):
```

Comment:
I would rather see it indented per this example from PEP 8:
```python
# Add some extra indentation on the conditional continuation line.
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```

But it is not a blocker for me, just a nit-pick :)

Reply:
Looks more readable to me too :)